### PR TITLE
:tada: Charlie Team Challenge 3 Solution

### PR DESCRIPTION
**Pull request by JakeCulp**

### CODE REVIEW

**Comment by philsinatra** [comment]
Looks like you've added a `package.json` file, but it does not include your dependencies information.

**Comment by philsinatra** [comment]
Looks good now. You'll want to update your instructions in the README file. Not only do we run the `gulp` command, but before doing that, there needs to be a `npm install` command to install dependencies.

**Comment by JakeCulp** [comment]
Not sure how I could combine the minifying of the css file and the copying of the html file into the same task.

File: challenge_3/charlie/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# How to run
+
+1. Open this folder inside of terminal
```
Comment (philsinatra) [inline challenge_3/charlie/README.md]: ~~Open this folder inside of terminal~~

1. `cd` to project root folder

**Comment by philsinatra** [comment]
Stylus and sourcemaps = 1 task
HTML move = 1 task

I'd change your instructions even more.

## Setup

If you do not have `gulp-cli` globally installed on your system....

```bash
# some helpful commands for installing gulp-cli
```

Once you have `gulp-cli` installed...

```bash
# more commands and instructions```

Make your instructions very clear and organized, as if written for a new developer who has no idea how to work with your task setup.

**Pull request merged**